packages/create-app/templates/default-app/packages/backend/src/plugins/search.ts.hbs

```diff
@@ -16,7 +19,14 @@ export default async function createPlugin({
   tokenManager,
 }: PluginEnvironment) {
   // Initialize a connection to a search engine.
+  {{#if dbTypeSqlite}}
   const searchEngine = new LunrSearchEngine({ logger });
+  {{/if}}
+  {{#if dbTypePG}}
+  const searchEngine = (await PgSearchEngine.supported(database))
+    ? await PgSearchEngine.from({ database })
+    : new LunrSearchEngine({ logger });
```

Comment:
Kinda wondering if we should simply just always install it with the fallback instead? It's not like there are any other DB options anyway x)

Reply:
Yeah, I had similar thoughts while doing this.  My thinking is: PG search will likely fulfill the needs of a certain niche of adopters, but that it's probably not the right choice for many or perhaps most, long-term.  So we don't want to encourage it _that_ much.  But respecting the `create-app` flag seems reasonable?  🤷‍♂️ 

We can tweak again based on feedback.